docs/tutorials/1_the_ibmq_account.ipynb

```diff
@@ -125,7 +125,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "where we have assumed that the user has stored their IQX account information locally ahead of time using `IBMQ.save_account(TOKEN)`. `TOKEN` here is the API token you obtain from your IQX account.\n",
+    "where we have assumed that the user has stored their IQX account information locally ahead of time using `IBMQ.save_account(TOKEN, HUB, GROUP, PROJECT)`. `TOKEN` here is the API token you obtain from your IQX account. `HUB`, `GROUP`, and `PROJECT` are optional parameters that allow you to specify a default provider you would like to save, along with your IQX account. This provider will be returned when loading your account.\n",
```

Comment:
Given the sample code at line 119 uses `IBMQ.save_account(TOKEN)`, it's probably better to not mention HUB/GROUP/PROJECT here to avoid confusion, especially since h/g/p was already explained at line 79.